operators/src/processing/time_shift.rs

```diff
@@ -24,8 +30,8 @@ pub type TimeShift = Operator<TimeShiftParams, SingleRasterOrVectorSource>;
 pub enum TimeShiftParams {
     /// Shift the query rectangle relative with a time step
     Relative {
-        step: TimeStep,
-        direction: RelativeShiftDirection,
+        granularity: TimeGranularity,
```

Comment:
also adjust in docs.geoengine.io, or if there is no doc for this operator: create it

Reply:
[There is already a task defined for this.](https://trello.com/c/QXjgg0Pe/808-ui-dialog-und-docs-f%C3%BCr-timeshift)